### PR TITLE
feat: New User model - column BALANCE added

### PR DESCRIPTION
**Pull request by FedericoDG**
## TICKET

- Modelo User - Agregar columna "balance"

## Description
AS: Developer
I Want: Add new column "balance"
To: Persist the user balance

##  Evidence:

NEWUSER

OLDUSER

### CODE REVIEW

File: src/database/migrations/20221102193311-create-user.js [hunk 1]
```diff
@@ -29,14 +29,20 @@ module.exports = {
       roleId: {
         type: Sequelize.INTEGER
       },
+      status: {
+        type: Sequelize.ENUM('active', 'blocked')
+      },
+      balance: {
+        type: Sequelize.DOUBLE
```
Comment (LucaGelmini) [inline src/database/migrations/20221102193311-create-user.js]: type: Sequeliz.Decimal(10,2)

File: src/database/models/user.js [hunk 2]
```diff
@@ -12,21 +12,26 @@ module.exports = (sequelize, DataTypes) => {
...
+    {
+      firstName: DataTypes.STRING,
+      lastName: DataTypes.STRING,
+      email: DataTypes.STRING,
+      password: DataTypes.STRING,
+      address: DataTypes.STRING,
+      avatar: DataTypes.STRING,
+      roleId: DataTypes.INTEGER,
+      status: DataTypes.ENUM('active', 'blocked'),
+      balance: DataTypes.DOUBLE
```
Comment (LucaGelmini) [inline src/database/models/user.js]: DataTypes.Decimal(10,2)

**Review by LucaGelmini — CHANGES_REQUESTED** [review]
balance type change to Decimal(10,2)

File: src/database/migrations/20221102193311-create-user.js [hunk 3]
```diff
@@ -33,7 +33,7 @@ module.exports = {
         type: Sequelize.ENUM('active', 'blocked')
       },
       balance: {
-        type: Sequelize.DOUBLE
+        type: Sequeliz.Decimal(10, 2)
```
Comment (LucaGelmini) [inline src/database/migrations/20221102193311-create-user.js]: Sequelize.Decimal(10, 2)

**Review by LucaGelmini — CHANGES_REQUESTED** [review]

File: src/database/migrations/20221102193311-create-user.js (hunk 3 above)
Comment (FedericoDG) [reply src/database/migrations/20221102193311-create-user.js]: Corregido.

**Review by LucaGelmini — APPROVED** [review]

**Pull request merged**